src/commands/slowlog-get.json

```diff
@@ -25,9 +25,9 @@
         "reply_schema": {
             "type": "array",
             "description": "Entries from the slow log in chronological order.",
+            "uniqueItems": true,
             "items": {
                 "type": "array",
-                "uniqueItems": true,
```

Comment:
members of subarray can be same. e.g timestamp and the amount of time needed for the execution